cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/TopicContentsService.java

```diff
@@ -18,6 +19,8 @@
 public class TopicContentsService {
 
   public static final String CUSTOM_OFFSET_SELECTION = "custom";
+  public static final String RANGE_OFFSET_SELECTION = "range";
```

Comment:
how abt having an enum for these two custom/range ?

Reply:
done